### PR TITLE
Added text about the gap analysis document, and the position of this …

### PR DESCRIPTION
**Pull request by r12a**
…document in the wider eco-system.

### CODE REVIEW

**Comment by r12a** [comment]
This addressed my action in https://github.com/w3c/alreq/issues/158

File: index.html [hunk 1]
```diff
@@ -157,7 +157,24 @@ <h3><a href="#h_about_this_document">About this document</a>
       </h3>
 
 
-      <p>Some text goes here.</p>
+      <p>The aim of this document is to describe the basic requirements for Arabic script layout and text support on the Web and in eBooks. These requirements provide information for Web technologies such as CSS, HTML and digital publications, and for application developers, about how to support users of Arabic scripts. The document focuses on Standard Arabic and Persian.</p>
+    </section>
+    
+    
+  <section id="h_gap_analysis">
+  <h3>Gap analysis</h3>
```
Comment (behnam) [inline index.html]: I would prefer "Gap Analysis" and "Other related resources" to be sub-sections of "About This Document", as they do provide background info on it.

Btw, IIUC, we capitalize h4 and up.

**Review by behnam — COMMENTED** [review]
Thanks, @r12a. Looks great! Only one little inline comment.

**Comment by r12a** [comment]
Ok @behnam, made your change.  Anyone object to me merging?

**Pull request merged**